### PR TITLE
Publish v1.17.13 and v1.18.5. [release]

### LINKED ISSUE #185: go1.18.5 is out
https://github.com/golang/go/issues?q=milestone%3AGo1.18.5+label%3ACherryPickApproved

https://groups.google.com/g/golang-announce/c/YqYYG87xB10

### PR DESCRIPTION
**Pull request by felicianotech**
https://twitter.com/golang/status/1554127315189354498

Closes #185

### CODE REVIEW

File: 1.18/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN sudo apt-get update && sudo apt-get install -y --no-install-recommends \
 
 # Install related tools
 ENV GOTESTSUM_V=1.8.1
-ENV GOCI_LINT_V=1.46.2
+ENV GOCI_LINT_V=1.45.2
```
Comment (JalexChen) [inline 1.18/Dockerfile:25]: is this supposed to go backwards?

Comment (felicianotech) [reply 1.18/Dockerfile:25]: Probably not. Let me check.

Comment (felicianotech) [reply 1.18/Dockerfile:25]: I see what happened. The change went into the Dockerfile and not the template. I screwed up my review for this PR: https://github.com/CircleCI-Public/cimg-go/pull/183

I'll get this done in a different PR, and then fix this PR. Great catch!

Comment (felicianotech) [reply 1.18/Dockerfile:25]: https://github.com/CircleCI-Public/cimg-go/pull/187

**Review by JalexChen — APPROVED** [review]

**Pull request merged**